packages/resource-deployment/templates/vnet.template.json

```diff
@@ -0,0 +1,57 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "type": "string",
+            "defaultValue": "[concat('vnet-a11y',toLower(uniqueString(resourceGroup().id)))]",
+            "metadata": {
+                "description": "The name of the Virtual Network"
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "The geo location for the vnet"
+            }
+        },
+        "addressPrefix": {
+            "type": "string"
+        },
+        "subnetName": {
+            "type": "string",
+            "defaultValue": "default"
+        },
+        "subnetAddressPrefix": {
+            "type": "string"
+        },
+        "enableDdosProtection": {
+            "type": "bool",
+            "defaultValue": false
```

Comment:
Will this enable basic DDoS level?

Reply:
Yes. I don't think there is a way to turn off DDoS protection altogether.